### ISSUE TITLE
validate date 'D:yyyymmddhhmmss-hh'

### ISSUE THREAD

**Issue by qntalfan** [pdfcpu/pdfcpu#341]
Built pdfcpu this morning from cloned github repo, on Fedora
workstation 34, binary in $GOPATH/bin, in bin directory executed:

./pdfcpu  properties list -v /home/dhmedley/Climate\ Change/Oregon\
Drought/20210601_west_trd.pdf

the validate command produces the same error output.

infile is directory where I store the weekly Drought Monitor report
from:

https://droughtmonitor.unl.edu/CurrentMap/StateDroughtMonitor.aspx?West

and I create the pdf file by clicking on the pdf icon from the "Legend
and statistics table:".

My thanks for any help you can give me on this; I am trying to produce
an animation of the drought severity using go and go-gtk (
https://github.com/mattn/go-gtk) and installed pdfcpu to extract images
from the saved *-trd.pdf files.
[crash.log](https://github.com/pdfcpu/pdfcpu/files/6601767/crash.log)
[20210601_west_trd.pdf](https://github.com/pdfcpu/pdfcpu/files/6601768/20210601_west_trd.pdf)

**Comment by hhrutter** [comment]
This will be fixed with the upcoming release.
The real issue was a bug in the date validation.
Thanks for reporting this 💚 
 
Unfortunately these files don't contain any images per se to extract.
The graphics are all rendered via PDF operators within the content stream.

**Comment by qntalfan** [comment]
My thanks -- now to accomplish my original task I will have to learn how to filter the pdf operators used and just recreate the map portion.
My teachable moment for the day, appreciated you pointing this out.

**Comment by hhrutter** [comment]
That's an interesting task!
Have fun 💚

**Comment by hhrutter** [comment]
Any reason why you don't use the images you can download at MAP DOWNLOAD no text on the left side of that page?

**Comment by henrixapp** [comment]
We have found/encountered a similar bug. Our PDF (edited with Xournal) errors while validating with:
```
validation error (obj#:24): pdfcpu: validateDateObject: <D:20210515103719+02'00> invalid date
```
We stumbled upon this error while testing the watermarking feature with an arbitrary pdf on my pc.
We would like to ignore the error, but as far as we know, this is not possible during watermarking (it does not produce an output.pdf with the xournal files).
